webfiling/config/routes/10-webfiling-seclogin.json.tpl

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
Just a thought, but this is the main webfiling route page now, rather than just for `seclogin`. Could rename it?

Reply:
Good point! I'll do that now